keyboards/handwired/prkl30/keymaps/default/keymap.c

```diff
@@ -0,0 +1,106 @@
+#include QMK_KEYBOARD_H
+#include "keymap_nordic.h"
+enum layers {
+  _DEFAULT,
+  _LOWER,
+  _RAISE,
+  _FN
+};
+
+enum custom_keycodes {
+    PRKL = SAFE_RANGE,
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+#define FN MO(_FN)
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Default
+ * ,-----------------------------------------------------------------------------------.------.
+ * | Esc  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  |  ¨^  | Bksp |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|------|
+ * |  Tab |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   Ö  |  Ä   | Entr |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|------'
+ * | Shift|   Z  |   X  |   C  |   V  |    Space    |   B  |   N  |   M  |   ,  |   .  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------'
+ */
+[_DEFAULT] = LAYOUT(
+    KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,
+    KC_LCTL,  KC_A,   KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    LT(_LOWER, KC_SPC),  _______,    KC_B,    KC_N,    KC_M,
+    KC_P,    NO_QUOT, KC_BSPC, NO_AE,   NO_OSLH, LT(_FN, KC_ENT), KC_COMM, LT(_RAISE, KC_DOT)
```

Comment:
These changes go with the suggestion to `prkl30.h`.

```suggestion
[_DEFAULT] = LAYOUT_all(
    KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,               KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    NO_QUOT, KC_BSPC,
    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,               KC_H,    KC_J,    KC_K,    KC_L,    NO_AE,   NO_OSLH, LT(_FN, KC_ENT),
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    LT(_LOWER, KC_SPC), _______, KC_B,    KC_N,    KC_M,    KC_COMM, LT(_RAISE, KC_DOT)
```